app/population/controllers/index.js

```diff
@@ -0,0 +1,4 @@
+const dashboard = require('./dashboard')
+module.exports = {
```

Comment:
Minor but could add a new line to separate these out like the rest of the codebase. I thought this was actually defined in the eslint rules.